src/net/sourceforge/kolmafia/request/AdventureRequest.java

```diff
@@ -1240,6 +1241,44 @@ private static String handleIntergnat(String monsterName) {
     return monsterName;
   }
 
+  public enum Dinosaur {
+    ARCHELON("archelon", "glass-shelled archelon that consumed "),
+    CHICKEN("chicken", "primitive chicken that just ate "),
+    DILOPHOSAUR("dilophosaur", "carrion-eating dilophosaur that consumed "),
+    FLATUSAURUS("flatusaurus", "steamy flatusaurus that recently devoured "),
```

Comment:
I think this can be any element word

Reply:
And those words would be ...?

Reply:
I added the ability for dinosaurs to have elements.
The only one I know - and have a test for - is a hot flatusaurus.

Reply:
Yendor and wRAR gave me the rest of the adjectives.

Reply:
Sorry - I added this comment from my bed past midnight and promptly fell asleep!